util/travis_utils.sh

```diff
@@ -10,8 +10,8 @@ TRAVIS_COMMIT_RANGE="${TRAVIS_COMMIT_RANGE:-HEAD~1..HEAD}"
 #TRAVIS_COMMIT_RANGE="c287f1bfc5c8...81f62atc4c1d"
 
 # Extra variables
-LOCAL_BRANCH=${TRAVIS_PULL_REQUEST_BRANCH:-${TRAVIS_BRANCH}}
-QMK_CHANGES=$(git diff --name-only HEAD ${TRAVIS_BRANCH})
+LOCAL_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+QMK_CHANGES=$(git diff --name-only -n 1 ${TRAVIS_COMMIT_RANGE})
```

Comment:
For the record/future reference, I can't actually find any indication of what `-n 1` does here.

Reply:
https://git-scm.com/docs/git-log#Documentation/git-log.txt--nltnumbergt 😉

Reply:
I thought it might be that but for some reason I couldn't find the reference. Your google-fu is much better than mine.

It also seems to run differently depending on whether you pass SHA hashes or branch names, at least when I try it locally. If I use branch names, `-n #` doesn't seem to work – running `git diff --name-only -n 1 master~2...master` still gives me the altered files from the previous two commits. I must be misunderstanding something somewhere. :slightly_frowning_face: